include/admin.h

```diff
@@ -26,3 +26,5 @@ bool admin_flush(PgSocket *admin, PktBuf *buf, const char *desc) /* _MUSTCHECK *
 bool admin_ready(PgSocket *admin, const char *desc)  _MUSTCHECK;
 void admin_handle_cancel(PgSocket *client);
 void admin_cleanup(void);
+PgSocket *find_client_global(const char *arg);
+PgSocket *find_socket_in_list(const char *arg, struct StatList *sockets);
```

Comment:
These two functions can be `static` afaict. When that's changed they wouldn't need to be in the header file.

Reply:
Fixed this in most recent commit.